test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -3010,9 +3041,11 @@ public void testGetDocContentListItemWithListAsParent() throws Exception {
             .replaceInContent(
               "ows_ScopeId='2;#{2E29615C-59E7-493B-B08A-3642949CC069}'",
               "ows_ScopeId='2;#{f9cb02b3-7f29-4cac-804f-ba6e14f1eb39}'"))
-        .register(new URLSegmentsExchange(
-            "http://localhost:1/sites/SiteCollection/Lists/Custom List/2_.000",
-          true, null, null, "{6F33949A-B3FF-4B0C-BA99-93CB518AC2C0}", "2"));
+        .register(new URLSegmentsExchange.Builder().setStrURL(
+            "http://localhost:1/sites/SiteCollection/Lists/Custom List/2_.000")
```

Comment:
Mostly not your doing, but I feel like every possible indentation style has been used in these expressions. It might be cleared up just by the pattern of one expression per line, with setter arguments indented +4.